dlt/common/storages/load_package.py

```diff
@@ -328,12 +328,20 @@ def complete_job(self, load_id: str, file_name: str) -> str:
     #
 
     def create_package(self, load_id: str) -> None:
-        self.storage.create_folder(load_id)
+        self.storage.create_folder(load_id, exists_ok=True)
```

Comment:
I am not sure wether by allowing this we will run into problems down the line. I can also try to make this more sensitive to only be allowed for normalize load packages. Or maybe I am also overlooking something else here..

Reply:
Can this affect incremental loads?

Reply:
@sultaniman not more than a regular loads. in the normalization stage `dlt` deals only with files, format etc. the extraction logic does not matter

@sh-rp when I look at the code, it looks like that when new package is created it is always initially "private" for a given step. (in new state). so we do not need owner!

also we do not need exits_ok -> it is better we catch those double create errors. please see my other comment